### DISCUSSION TITLE
Update recommended prior to iOS 18.1 general availability

### DISCUSSION THREAD

**Discussion by Kyle-Reis** [microsoftconnect/ms-intune-app-sdk-ios#477] [Announcements]
The iOS 18.1 Beta 5 release contained a breaking change which impacts the iOS Intune SDK's ability to block the iOS 18 writing tools feature when app protection policy is configured to restrict outgoing data transfer. This issue has been addressed in the [19.7.1 release](https://github.com/microsoftconnect/ms-intune-app-sdk-ios/releases/tag/19.7.1) for Xcode 15 and the [20.1.2 release](https://github.com/microsoftconnect/ms-intune-app-sdk-ios/releases/tag/20.1.2) for Xcode 16. We recommend that all application developers upgrade to the latest available supported release which corresponds to the Xcode version your application is currently compiled with, and that you push your updated application to production as soon as possible to ensure the fix for this issue is available before iOS 18.1 becomes generally available. If you have any questions, please [file an issue](https://github.com/microsoftconnect/ms-intune-app-sdk-ios/issues) in this repository or reply directly to this announcement.

**Comment by Regina-China** [comment]
**Sharing of more details of this new release change.**

The issue is related only to the "Send org data to other apps" setting. When this setting is configured to only allow sharing to policy managed app or to block sharing to other applications altogether, the MAM SDK should block the use of the "Writing Tools" context menu option that sometimes appears when selecting text on iOS 18.0+
 
However, because of the breaking change Apple made in iOS 18.1 Beta 5, the original blocking functionality no longer worked, and we issued a fix in the 19.7.1 and 20.1.2 Intune SDK releases for Xcode 15 and 16, respectively
 
If the application does not contain one of those versions of Intune (or a higher minor/patch version of the 19.x and 20.x releases), then it will not successfully block the use of Writing Tools when policy is configured to do so
 
Writing tools are not a standalone app, but it runs outside of the host app in a separate process, where Intune cannot protect the data.

**Comment by leoslima1** [comment]
@Kyle-Reis how about the bindings libraries built for MAUI?
Microsoft.Intune.Essentials.iOS and Microsoft.Intune.Essentials.Android, are these libraries already updated to this new version?

**Reply by Kyle-Reis** [reply]
@leoslima1[ Microsoft.Intune.Essentials.iOS bindings](https://www.nuget.org/packages/Microsoft.Intune.Maui.Essentials.iOS) for .Net/MAUI do not yet support Xcode 16 (primarily because Xcode 16 support for .Net/MAUI itself was only recently added), so no 20.x release is available there yet. However, that will soon change.

The 19.7.1 release for Xcode 15 is currently available for .Net/MAUI.

This issue is iOS specific and so it would not apply to the Microsoft.Intune.Essentials.Android binding.
